.run/run.sh

```diff
@@ -0,0 +1,7 @@
+# This is the root directory of datadog agent project as set in the vagrant script
```

Comment:
#! /usr/bin/bash -ex

Reply:
the script can be run on bare linux machine and thus the root dir does not match everyone's local repo

You can use `$(dirname $(dirname $(realpath "$0")))` or `$(git rev-parse --show-toplevel)`

Reply:
Same as previous comment :)